templates/error.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'base.htm' %}
+
+{% block content %}
+    <!-- This template was made by Colorlib (https://colorlib.com)
+         Please visit our website for more awesome templates, themes and tools. -->
+	<div id="notfound">
+		<div class="notfound">
+			<div class="notfound-404">
+				<h1>Oops!</h1>
+				<h2>
+                    {% block message %}{% endblock %}
+                </h2>
+			</div>
+			<a href="/">Go TO Homepage</a>
+		</div>
+	</div>
+{% endblock content %}
+
+{% block styles %}
+    <meta charset="utf-8">
```

Comment:
Let's remove these meta tags.  

The `styles` block is just for CSS (although we could add another block in base.htm for additional meta tags, if needed). The `base.htm` template already has the `charset` and `viewport` meta tags:
```
<meta charset="UTF-8" />
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
As for the http-equiv meta tag, it's probably not necessary, since we probably don't need to worry about supporting IE 8/9 (see https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do).